pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
+
+`printf "header1 header2\nbar foo\n" | column -c {{10}}`
+
+- Specify characters to use to delimit columns for the -t option (i.e. "," for csv), default is whitespace:
+
+`printf "header1,header2\nbar,foo\n" | column -s{{,}}`
+
+- Determine the number of columns and auto-align the output in a tabular format:
```

Comment:
Well, the "automatically" refers to the "Determine the number of columns".
The other part was a proposal of @agnivade and I find if nice.
This is the original man page, how would you rewrite that?
```
-t
              Determine the number of columns the input contains and create
              a table.  Columns are delimited with whitespace, by default,
              or with the characters supplied using the --output-separator
              option.  Table output is useful for pretty-printing.
```